content/docs/command-reference/import-url.md

```diff
@@ -125,12 +125,13 @@ source.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
-- `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
-  the file or directory already exists, this can be used to skip the download.
-  The data hash is not calculated when this option is used, only the import
-  metadata is saved to the `.dvc` file. `dvc commit <out>.dvc` can be used if
-  the data hashes are needed in the `.dvc` file, and to save existing data to
-  the cache.
+- `--no-exec` - create the import `.dvc` file but don't download `url`. The No
```

Comment:
The No

Reply:
Still think that `No hash values are calculated or saved to the `.dvc` file.` will be very hard to understand. We don't even mention anywhere else about any hash calculations.

Can we describe this "no hash" using some human friendly message - like .dvc file is half-baked, or .dvc file is detached ... or .dvc file is a placeholder - URL is not checked, data is not downloaded, thus we don't save any information about it.

Reply:
> We don't even mention anywhere else about any hash calculations.

Good point. I guess we can skip this note completely (implementation detail). Updated this and related files where the same problem existed.

> some human friendly message - like .dvc file is half-baked

I think that the next sentence, "Use `dvc update` to finish the operation later", implies that it's half-baked.